CMSgov/onemac
# OY2-2467 Allow OKTA login to work with custom domains

Changes:
1. Added generation of application URL based on either custom domain if it exists or the default AWS provided domain if not.
2. Updated some of the documentation, but it is still incomplete.  Will continue populating it as we go, but focusing on bug fix.

Test steps:
Unfortunately we can only test one half of the change in our feature branches or develop, and the other half must be tested in VAL.
1. Access the endpoint and verify that the application is shown.

Endpoint: https://d1108mqszec0hp.cloudfront.net
